data/Tunisia/Majlis/ep-popolo-v1.0.json

```diff
@@ -4796,7 +4796,7 @@
       "name": "Dorra Yaacoubi"
     },
     {
-      "birth_date": "1970-04-30",
+      "birth_date": "1970",
```

Comment:
when the two wikidata pages were merged, it ended up with two birth dates. I've removed the one that's only at year specificity.